Tinylog/Views/Cells/TaskTableViewCell.swift

```diff
@@ -7,28 +7,33 @@
 //
 // swiftlint:disable force_unwrapping
 import UIKit
-import TTTAttributedLabel
+import Nantes
 
 final class TaskTableViewCell: GenericTableViewCell {
 
     let kLabelHorizontalInsets: CGFloat = 60.0
     let kLabelVerticalInsets: CGFloat = 10.0
-    let taskLabel: TTTAttributedLabel = TTTAttributedLabel(frame: CGRect.zero)
+    let taskLabel: NantesLabel = NantesLabel(frame: CGRect.zero)
 
     let checkBoxButton: CheckBoxButton = CheckBoxButton()
     var checkMarkIcon: UIImageView?
     var managedObjectContext: NSManagedObjectContext!
 
-    var currentTask: TLITask? {
+    var task: TLITask? {
         didSet {
-
+            guard let task = task,
+                let list = task.list,
+                let color = list.color else {
+                return
+            }
+            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/7d74fdd396913e4e42ba0446eaceaffef14d4bd5/Tinylog/Views/Cells/TaskTableViewCell.swift#L29"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TaskTableViewCell.swift:29</code></td>
    </tr>
  </tbody>
</table>